plugins/teststeps/sshcmd/sshcmd.go

```diff
@@ -186,31 +201,39 @@ func (ts *SSHCmd) Run(cancel, pause <-chan struct{}, ch test.TestStepChannels, p
 			errCh <- innerErr
 		}()
 
-		select {
-		case err := <-errCh:
-			log.Infof("Stdout of command '%s' is '%s'", cmd, stdout.Bytes())
-			if err == nil {
-				// Execute expectations
-				expect := ts.Expect.String()
-				if expect == "" {
-					log.Warningf("no expectations specified")
-				} else {
-					re := regexp.MustCompile(expect)
-					matches := re.FindAll(stdout.Bytes(), -1)
-					if len(matches) > 0 {
-						log.Infof("match for regex \"%s\" found", expect)
+		for {
+			select {
+			case err := <-errCh:
+				log.Infof("Stdout of command '%s' is '%s'", cmd, stdoutBuf.Bytes())
+				if err == nil {
+					// Execute expectations
+					expect := ts.Expect.String()
+					if expect == "" {
+						log.Warningf("no expectations specified")
 					} else {
-						return fmt.Errorf("match for %s not found for target %v", expect, target)
+						re := regexp.MustCompile(expect)
+						matches := re.FindAll(stdoutBuf.Bytes(), -1)
+						if len(matches) > 0 {
```

Comment:
what is this `stdoutBuf` stuff doing here, I assume it leaked from one of the other PRs?

Reply:
Yes that should not be here. Let me fix this.